src/fsrs/default.ts

```diff
@@ -5,47 +5,47 @@ import { clamp } from './help'
 
 export const default_request_retention = 0.9
 export const default_maximum_interval = 36500
-export const default_w = [
+export const default_w = Object.freeze([
```

Comment:
```suggestion
export const default_w = [
```
Just to check you are aware of this `Object.freeze` right, I don't want an extra major version for this I'm just saying it would have to be one because editing this array before would cause a runtime error after the update and not before.